### PR TITLE
Run on Symfony 3.1

### PR DESCRIPTION
**Pull request by WouterJ**
We're finally there: The CMF sandbox is now able to run on Symfony 3(.1).

### CODE REVIEW

**Comment by dbu** [comment]
i just merged #328... can you rebase this one? something is up with styleci.

**Comment by WouterJ** [comment]
PR is now rebased

File: app/AppKernel.php [hunk 1]
```diff
@@ -37,8 +37,9 @@ public function registerBundles()
             new Symfony\Cmf\Bundle\TreeBrowserBundle\CmfTreeBrowserBundle(),
             new Symfony\Cmf\Bundle\BlockBundle\CmfBlockBundle(),
             new Symfony\Cmf\Bundle\SeoBundle\CmfSeoBundle(),
-            new Burgov\Bundle\KeyValueFormBundle\BurgovKeyValueFormBundle(),
```
Comment (dbu) [inline app/AppKernel.php]: afaik we need this in sonata admin

Comment (WouterJ) [inline app/AppKernel.php]: it's moved below to the "admin bundle" section

File: composer.json [hunk 2]
```diff
@@ -10,52 +10,57 @@
...
         "doctrine/data-fixtures": "^1.0",
         "doctrine/doctrine-cache-bundle": "^1.0",
         "doctrine/phpcr-odm": "^1.3",
 
         "sonata-project/cache-bundle": "^2.1",
-        "sonata-project/doctrine-phpcr-admin-bundle": "1.2.*",
-        "sonata-project/admin-bundle": "^2.3.3",
-        "sonata-project/translation-bundle": "1.0.0|^1.0.2",
+        "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
+        "sonata-project/admin-bundle": "dev-fix-bug-admin-groups as 3.1.0",
```
Comment (dbu) [inline composer.json]: https://github.com/sonata-project/SonataAdminBundle/pull/3903 has been merged.

not sure why this fails in install, you do tell composer to look at your fork of the admin-bundle repo...

Comment (WouterJ) [inline composer.json]: I deleted my branch after merge, but forgot to revert this.

**Comment by dbu** [comment]
:tada: 

thanks a lot! i guess we wait with tagging anything until the dependencies are stable and we  no longer need the minimum_stability option in composer.json

**Pull request merged**